readme.md

```diff
@@ -207,6 +207,10 @@ If the file extension is omitted, it is auto-completed to the correct extension
 
   Whether SSL certificates are required to be valid when downloading Electron. **Defaults to `true`**.
 
+  `tmpdir` - *String*
```

Comment:
Minor nit - this is the wrong indentation. There should be no spaces between the beginning of the line and the option name. The description's indentation is fine.